### ISSUE TITLE
Click and drag? 

### ISSUE THREAD

**Issue by user928** [terrymun/paver#11]
Is it possible to have click and drag?

**Comment by terrymun** [comment]
Probably yes.

**Comment by user928** [comment]
Is it already supported with some parameter?

**Comment by terrymun** [comment]
No, it isn't. You'll probably have to hack your way around it. Like by using a null smoothing function, and manually setting the pan values depending on mouse position.

**Comment by user928** [comment]
Ohh ok, so this one can go to 'wish list' tasks ;)
